parquet-tools/src/test/java/org/apache/parquet/tools/read/TestSimpleMapRecord.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.parquet.tools.read;
```

Comment:
Looks like you forgot to add a license header to this file and that's causing Travis CI to fail.